performance_test/src/experiment_execution/analyze_runner.cpp

```diff
@@ -57,6 +57,7 @@ AnalyzeRunner::AnalyzeRunner()
 
 void AnalyzeRunner::run() const
 {
+  static bool is_first_entry = true;
```

Comment:
This must be a member variable as this code will break if there are multiple instances of `AnalyzeRunner`.

Reply:
Make sense. Made it as a member variable